lib/createFormatter.js

```diff
@@ -37,6 +39,8 @@ module.exports = function createFormatter(newrelic) {
     }
     info.timestamp = Date.now()
 
+    const metadata = newrelic.getLinkingMetadata(true)
```

Comment:
Looks like we're not checking for undefined as part of this change, but that seems fine since [the method](https://github.com/newrelic/node-newrelic/blob/2dd871030f248de58e7cb4df5da4736f77cc15bd/api.js#L151) should always return an object

Reply:
Older versions did not return an object (thus the check) but we'll never get this far due to the new check on the `shim` existing to confirm we have a fully functional agent.

The shim check isn't ideal, eventually I'd like something more concrete.. but it doesn't require overhauling anything.